tests/Oryx.BuildImage.Tests/Node/NodeJsSampleAppsTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.Oryx.BuildScriptGenerator.Common;
+using Microsoft.Oryx.BuildScriptGeneratorCli;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.BuildImage.Tests
+{
+    public class NodeJsSampleAppsTest : SampleAppsTestBase
```

Comment:
Interesting. Isn't there already a file and class with this name? Also I would rename this file and class to indicate these are mono repo related tests